media/css/firefox/firstrun/firstrun-quantum.scss

```diff
@@ -53,115 +57,31 @@ body {
     width: 100vw;
 }
 
-.background {
-    left: 0;
-    margin: auto;
-    position: fixed;
-    right: 0;
-    transform: scale(1.5);
-}
-
-.white-overlay,
-.white-background {
-    fill: #FFFFFF;
-    height: 100vh;
-    position: absolute;
-    width: 100vw;
-}
-
-.white-overlay {
-    opacity: 0;
-    will-change: opacity;
-}
-
-.wave1 {
-    fill: url('#light-gradient');
-    height: 100vh;
-    width: 100vw;
-}
-
-.wave2 {
-    bottom: 0;
-    fill: url('#dark-gradient');
-    height: 70vh;
-    width: 85vw;
-}
-
-.wave3 {
-    bottom: 0;
-    fill: url('#dark-gradient');
-    height: 48vh;
-    opacity: 0.5;
-    width: 75vw;
-}
-
-.wave4 {
-    bottom: 0;
-    fill: #005FE7;
-    height: 26vh;
-    opacity: 0.6;
-    width: 100vw;
-}
-
-.wave5 {
-    fill: #002079;
-    height: 51vh;
-    opacity: 0.6;
-    width: 100vw;
-}
-
-.wave6 {
-    fill: url('#light-gradient');
-    height: 73vh;
-    opacity: 0.8;
-    width: 100vw;
-}
-
-.wave7 {
-    fill: url('#light-gradient');
-    height: 59vh;
-    margin-left: 0;
-    opacity: 0.3;
-    width: 60vw;
-}
-
-.wave8 {
-    fill: #002079;
-    height: 51vh;
-    opacity: 0.6;
-    width: 100vw;
-}
-
-.color-overlay {
-    fill: url('#orange-gradient');
-    height: 100vh;
-    mix-blend-mode:color;
-    opacity: 0;
-    position: absolute;
-    width: 100vw;
-    will-change: opacity;
-}
-
 /* -------------------------------------------------------------------------- */
 // Main page content container
 
 .fxaccounts-container {
     color: #fff;
     z-index: 10;
+    margin: auto;
 
-    @media #{$mq-medium} {
-        bottom: 0;
+  @media #{unquote($mq-medium)} {
         height: 424px;
-        left: 0;
         margin: auto;
-        position: absolute;
-        right: 0;
-        top: 0;
         width: 750px;
+        padding-top: 20px;
```

Comment:
This ends up with the content *pretty* tight to the top of the page.

Taking a step back, I'm wondering if we should just throw out this existing layout CSS. For example, after my PR adding the animated Sync image, the content now exceeds the `height: 424px` rule above. There's also `float`ing going on, which surely could be improved with grid.

Reply:
I'm game to grid-ify it. That might fix the problem with the iframe jumping around in height too.